File: abc/abc283/a.cpp
// https://atcoder.jp/contests/abc283/submissions/37512837

#include <bits/stdc++.h>
using namespace std;

int main() {
  cin.tie(0);
  ios_base::sync_with_stdio(0);

  int a, b, ans = 1;
  cin >> a >> b;
  while(b--) ans *= a;

  cout << ans;
}

File: abc/abc283/a.java
// https://atcoder.jp/contests/abc283/submissions/43222026

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();

    int ans = 1;
    for(int i = 0; i < b; i++) {
      ans *= a;
    }

    System.out.println(ans);
  }
}
